server/asset/btc/rpcclient.go

```diff
@@ -128,10 +131,43 @@ func (rc *RPCClient) GetRawTransactionVerbose(txHash *chainhash.Hash) (*btcjson.
 		true}, res)
 }
 
-// GetBlockVerbose fetches verbose block data for the specified hash.
+// GetBlock fetches raw block data, and the "verbose" block header, for the
+// block with the given hash. The verbose block header return is separate
+// because it contains other useful info like the height and median time that
+// the wire type does not contain.
+func (rc *RPCClient) GetBlock(blockHash *chainhash.Hash) (*wire.MsgBlock, *btcjson.GetBlockHeaderVerboseResult, error) {
+	arg := interface{}(0)
+	if rc.booleanVerboseGetBlock {
+		arg = false
+	}
```

Comment:
Just to ask, I'm sure you've considered this, doesn't the backend parse `0` as `false` or `false` as `0` as needed?

Reply:
bitcoind, yes, dogecoind, no.

```
$  ./dogecoin-cli -testnet getblock 64acaa2347aafe30e8898dbe3c7dd3cef118ba690c79dedc7b463a5c01b0ef8e 0
error code: -1
error message:
JSON value is not a boolean as expected
```

We could assume boolean would always work fine for bitcoind and clones, but now that we know we were glossing over this, we should be explicit.

Plus, there's a "2" if we ever want to do that, so we should have the integer nature of the arg front and center.

Reply:
https://github.com/decred/dcrdex/pull/1558#discussion_r848878728